gpflow/optimizers/scipy.py

```diff
@@ -136,10 +140,9 @@ def assign_tensors(to_tensors: Iterable[tf.Variable], values: Iterable[tf.Tensor
             tensor.assign(tensor_vector)
 
 
-V = TypeVar("V", bound=Variables)
-
-
-def _compute_loss_and_gradients(loss_closure: LossClosure, variables: V) -> Tuple[tf.Tensor, V]:
```

Comment:
This type says we return a list of variables as the second return value, but actually it's a list of tensors